march_rqt_gait_generator/test/nosetests/gait_generator_controller_test.py

```diff
@@ -24,6 +24,7 @@ def setUp(self):
         self.duration = self.gait_generator_controller.subgait.duration
         self.num_joints = len(self.gait_generator_controller.subgait.joints)
 
+        self.maxDiff = None
```

Comment:
What is this?

Reply:
That is an artifact that should be removed